pkg/commands/compute/compute_mocks_test.go

```diff
@@ -97,3 +97,16 @@ func (v versionClient) Do(req *http.Request) (*http.Response, error) {
 	}
 	return rec.Result(), nil
 }
+
+func createBackendExpect(address string, port uint, overrideHost string, sslSNIHostname string) func(*fastly.CreateBackendInput) (*fastly.Backend, error) {
```

Comment:
NIT: I'd prefer to have this code inside of `compute_deploy_test.go` until such a time that the specific behaviour is necessary to put here for sharing across other test code.

Reply:
Done.